src/main/java/hudson/tasks/Ant.java

```diff
@@ -104,7 +104,7 @@
     private final String properties;
     
     @DataBoundConstructor
-    public Ant(String targets,String antName, String antOpts, String buildFile, String properties) {
+    public Ant(String targets, String antName, String antOpts, String buildFile, String properties) {
```

Comment:
Unfortunately https://github.com/jenkinsci/ant-plugin/pull/33/files?w still does not ignore this, making this diff too long to be read.